HousingResidentInformationAPI/serverless.yml

```diff
@@ -15,7 +15,7 @@ functions:
     handler: HousingResidentInformationAPI::HousingResidentInformationAPI.LambdaEntryPoint::FunctionHandlerAsync
     role: lambdaExecutionRole
     environment:
-      CONNECTION_STRING: Host=${ssm:housing-api/${self:provider.stage}/postgres-hostname};Port=${ssm:housing-api/${self:provider.stage}/postgres-port};Database=uh_mirror;Username=${ssm:housing-api/${self:provider.stage}/postgres-username};Password=${ssm:housing-api/${self:provider.stage}/postgres-password}
+      CONNECTION_STRING: Host=${ssm:/housing-api/${self:provider.stage}/postgres-hostname};Port=${ssm:/housing-api/${self:provider.stage}/postgres-port};Database=uh_mirror;Username=${ssm:/housing-api/${self:provider.stage}/postgres-username};Password=${ssm:housing-api/${self:provider.stage}/postgres-password}
```

Comment:
I think the password also needs a leading `/`

Reply:
Good spot, I've now made that change